### ISSUE TITLE
Questions about training on VOCASET

### ISSUE THREAD

**Issue by ChenVoid** [uuembodiedsocialai/FaceDiffuser#8]
Overfitting happens when the model is trained on VOCASET. The training loss is descending while the validation loss is rising step by step when training.
So is VOCASET too small? Dose anyone meets the same problem?

**Comment by Starind** [comment]
I meet the same problem, and I don't know why

**Comment by YifengMa9** [comment]
Hi, I have retrained the model on VOCASET without overfitting. The performance is similar to the official checkpoint.

The script I used is:
```
python main.py --dataset vocaset --vertice_dim 15069 --feature_dim 256 --output_fps 30 --train_subjects FaceTalk_170728_03272_TA\ FaceTalk_170904_00128_TA\ FaceTalk_170725_00137_TA\ FaceTalk_170915_00223_TA\ FaceTalk_170811_03274_TA\ FaceTalk_170913_03279_TA\ FaceTalk_170904_03276_TA\ FaceTalk_170912_03278_TA --val_subjects FaceTalk_170908_03277_TA\ FaceTalk_170811_03275_TA --test_subjects FaceTalk_170809_00138_TA\ FaceTalk_170731_00024_TA --diff_steps 1000 --gru_dim 256
```

I think the sucessful retraining is owing to several changes: 
1. I change the `--gru_dim` to `256`. The `gru_dim` setting is different from the paper but identical to the released checkpoint. 
2. I make sure that the training, dev, and test set setting is identical to the paper. The setting should be specified by `--train/val/test_subjects`.
3. I fix some bugs in the script. The bugs come from the fact that the provided training script is for `BIWI`, not `vocaset`. To fix bugs, it is recommended to debug line by line.

My loss curve shows:

![losses_face_diffuser_vocaset](https://github.com/uuembodiedsocialai/FaceDiffuser/assets/9561442/b171159a-e63f-4504-b825-ccedae037733)

**Comment by HiouKaoru** [comment]
I followed the instructions to process the downloaded the training data of vocaset and obtained an incomplete sequence. For example, I am missing FaceTalk_170809_00138-TA_sentence32.npy. I would like to know if anyone has the same issue or if anyone knows how to solve it.

**Comment by paopao1027** [comment]
> Hi, I have retrained the model on VOCASET without overfitting. The performance is similar to the official checkpoint.
> 
> The script I used is:
> 
> ```
> python main.py --dataset vocaset --vertice_dim 15069 --feature_dim 256 --output_fps 30 --train_subjects FaceTalk_170728_03272_TA\ FaceTalk_170904_00128_TA\ FaceTalk_170725_00137_TA\ FaceTalk_170915_00223_TA\ FaceTalk_170811_03274_TA\ FaceTalk_170913_03279_TA\ FaceTalk_170904_03276_TA\ FaceTalk_170912_03278_TA --val_subjects FaceTalk_170908_03277_TA\ FaceTalk_170811_03275_TA --test_subjects FaceTalk_170809_00138_TA\ FaceTalk_170731_00024_TA --diff_steps 1000 --gru_dim 256
> ```
> 
> I think the sucessful retraining is owing to several changes:
> 
> 1. I change the `--gru_dim` to `256`. The `gru_dim` setting is different from the paper but identical to the released checkpoint.
> 2. I make sure that the training, dev, and test set setting is identical to the paper. The setting should be specified by `--train/val/test_subjects`.
> 3. I fix some bugs in the script. The bugs come from the fact that the provided training script is for `BIWI`, not `vocaset`. To fix bugs, it is recommended to debug line by line.
> 
> My loss curve shows:
> 
> ![losses_face_diffuser_vocaset](https://private-user-images.githubusercontent.com/9561442/305690176-b171159a-e63f-4504-b825-ccedae037733.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNTk4NDUsIm5iZiI6MTcxMjA1OTU0NSwicGF0aCI6Ii85NTYxNDQyLzMwNTY5MDE3Ni1iMTcxMTU5YS1lNjNmLTQ1MDQtYjgyNS1jY2VkYWUwMzc3MzMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MDJUMTIwNTQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzdjYTVhOGIxNTIzOGI4MWI2NjE0YTY5M2E0ZDIzYjQ0NTI2MDllODkxNjdhMWQyNzhjMjY0N2ExZTE1ZmFhNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.3ItfmywiY5UOryUbfOxRxP--jCKd2ZH60I-MVXa-VyI)

The validation set curve does not decrease as steadily as the training set curve, as shown in the graph. Doesn't this also count as overfitting

**Comment by csyufei** [comment]
where are these files, namely data_verts.npy, raw_audio_fixed.pkl, templates.pkl and subj_seq_to_idx.pkl in [https://voca.is.tue.mpg.de/download.php](url)? Thanks!
